docs/developer_guides/code_systems/feature_flags.md

```
@@ -29,7 +29,7 @@ flag.is_active_for_team(my_team)
 
 !!! tip
 
-    Flags are created automatically in the database when referenced in code or templates so it is not necessary to create them manually.
+    Flags are created automatically in the database when referenced in code or templates, so it is not necessary to create them manually.
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)